packages/react-next/src/components/Image/Image.test.tsx

```diff
@@ -18,6 +19,10 @@ describe('Image', () => {
     Object.defineProperty(HTMLImageElement.prototype, 'naturalWidth', { get: () => 1 });
   });
 
+  beforeEach(() => {
```

Comment:
It might be possible to just move this to config/tests.js
```
beforeAll(() => {
  resetIds()
});
```

https://jestjs.io/docs/en/setup-teardown#one-time-setup

Reply:
This will help, but if we truly avoid the noise, resetting id should be done before for each test run rather than each test file run.

Reply:
it's certainly a tradeoff. but i definitely like doing this once and never need to worry about it again.  🤔 i think i will take that